_posts/2019-04-25-is-cv-the-best-way-to-hire-a-software-developer.md

```diff
@@ -0,0 +1,65 @@
+---
+
+![Photo by Clem Onojeghuo on Unsplash](/images/cv-resume/featured.jpg)
+
+I really hate doing things I know are borderline bullshit just because they are conventional wisdom. CVs are definitely one of them. First of all, biases always find their way around a CV. Age, sex and name are just some of the information we now reduct or obfuscate in order to remove certain biases. I would argue we need to deduct the whole CV in order to be somewhat impartial and effective. But then, you might ask, how do we vet people if not through a CV? How would you know about their background, skills and experience if you don't read their CV?
+
+Let me save you some time. People exaggerate so what you see on a CV is a stretch of their actual experience. The sad thing is everyone knows this. But I'd like to examine a couple of *sections* on the CV bit closer that reflect my own biases when I'm reading someone's CV.
+
+### Education
+Cambridge University. I can hear myself thinking, "Cool this is a smart person".
+
+University of x. "Meh, I'm not sure about this university. I have heard it before; maybe it's good, maybe it's not".
+
+"Politecnico di Milano", I can hear my brain shouting "This is not even a British university! How do I even begin to find out if it's good or not?! I will not bother checking it out".
+
+Bootcamp. Well for sure they are not experienced enough. How could they possible know about garbage collection cycles". For the record, I have no idea about garbage collection cycles apart from that blog post I read 2 years ago on Hacker News. So, you can call me an expert I guess.
+
+So, 4 humans but only 1 seems to be getting a 🥇. Why is that? People often think that discounting university degrees is an attack on formal education. That's not the case. Maybe it's an attack on elitism actually. It's not about making candidate number 1 more but rather why are the other 3 candidates less?
+
+In our great and diverse industry, there are plenty of developers that didn't even study Computer Science. Some of them are the best I know. An example is my good friend [Txus](https://twitter.com/txustice).
+
+So how about we drop that section completely? It takes a bit of a real estate on paper as well.
+
+### Experience
+Fasten your seat belts for this section. Mind the gap as you enter the platform.
+
+When I was at Deliveroo, my CV will probably say, I worked on Kafka to decouple a monolith into microservices with a throughput of 1 billion messages a nanosecond. True story. Actually, partially true. I did work at Deliveroo. And I have used a thing called Kafka. It was provided as a library of course so I was able to call `Kafka.please_publish` and `Kafka.please_consume_without_dying_because_i_have_no_idea_what_i_am_doing`. I have no idea about the internals of Kafka, what partitions are, how to achieve ordering or how to replay events. *Even this is not necessarily true because I was involved with Kafka at carwow quite a bit 😅.*
```

Comment:
I think you need to explain the fact that you worked at `carwow` here, as it's just been randomly dropped here with no context.